playground/grid.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { useRef, useEffect, useCallback, useState } from 'react'
+import * as React from 'react'
+import { useRef, useEffect, useCallback, useState } from 'react'
```

Comment:
so far, I just changed as react core did in their migration work.

Essentially, it is importing same thing twice.
Because react team leave as it is, I guess it is fine.
  but, my personal preference is like bellow.

```suggestion
import * as React from 'react'

const { useRef, useEffect, useCallback, useState } = React
```

Reply:
another option would be 
```suggestion
import * as React from 'react'
```
and use `React.x`